packages/frontend/@n8n/i18n/src/locales/en.json

```diff
@@ -2368,6 +2369,21 @@
 	"workflowDetails.active": "Active",
 	"workflowDetails.addTag": "Add tag",
 	"workflowDetails.chooseOrCreateATag": "Choose or create a tag",
+	"workflowExtraction.error.failure": "Sub-workflow Extraction failed",
+	"workflowExtraction.error.selectionGraph.inputEdgeToNonRoot": "Non-input node '{node}' has a connection from outside the selection.",
+	"workflowExtraction.error.selectionGraph.outputEdgeFromNonLeaf": "Non-output node '{node}' has a connection to outside the selection.",
```

Comment:
"Non-input node {node} has a connection **to** a node outside the current selction."

Reply:
No, my mistake leave the `''`